src/lib/ngsi/ContextElementResponseVector.cpp

```diff
@@ -73,6 +73,31 @@ std::string ContextElementResponseVector::render
 
 /* ****************************************************************************
 *
+* ContextElementResponseVector::toJson - 
+*/
+std::string ContextElementResponseVector::toJson(ConnectionInfo* ciP, NotificationFormat notifyFormat)
```

Comment:
Same here. ciP is not used, except for passing down to CER toJson() (and, in CER toJson(), ciP is not used, as mentionced in a different comment).